### DISCUSSION TITLE
How to make the parent drug not inhibit its metabolite

### DISCUSSION THREAD

**Discussion by balamo98** [Open-Systems-Pharmacology/Forum#1439] [Questions & Problems] [answered]
Dear team，
I'm simulating a parent-metabolite PBPK model. When I conduct single drug simulation, the fitting is good. As I know, it's a weak CYP3A and P-gp inhibitor. So I added inhibition to it.
![图片](https://user-images.githubusercontent.com/96385989/212681077-f7cab84c-6bfa-40c0-afe3-501346da4edb.png)
But when I simulated it later, I found that the parent drug would inhibit its active metabolite.
The predicted concentration of its metabolites is too high and the fitting is not good. 
Add these inhibitions, but is it possible not to let it inhibition its metabolites.
Its metabolite is cleared by “total hepatic clearance”

Yours，colin

**Comment by StephanSchaller** [comment, marked as answer]
Hi @balamo98 , certainly, an Inhibition is only active per each process, if you strictly activate it when creating a simulation.
In the tab "processes" at the very top are listed your inhibitions. Just remove the one for the metabolite process.

But if you only have "total hepatic CL" for the metabolite, you should not even be able to define a DDI for this process. DDIs can only be defined for processes associated with proteins (enzymes, Transporters, ..).

**Reply by balamo98** [reply]
Thank you for your reply， I will modify my model.
